examples/oemof_0.2/basic_example/basic_example.py

```diff
@@ -0,0 +1,213 @@
+# imports
+###############################################################################
+
+# Default logger of oemof
+from oemof.tools import logger
+from oemof.tools import helpers
+
+import oemof.solph as solph
+from oemof.outputlib import processing, views
+
+import logging
+import os
+import pandas as pd
+import pprint as pp
+
+try:
+    import matplotlib.pyplot as plt
+except ImportError:
+    plt = None
+
+
+solver = 'cbc'  # 'glpk', 'gurobi',....
+debug = False  # Set number_of_timesteps to 3 to get a readable lp-file.
+number_of_time_steps = 24*7*8
+solver_verbose = False  # show/hide solver output
+
+# initiate the logger (see the API docs for more information)
+logger.define_logging(logfile='oemof_example.log',
+                      screen_level=logging.INFO,
+                      file_level=logging.DEBUG)
+
+logging.info('Initialize the energy system')
+date_time_index = pd.date_range('1/1/2012', periods=number_of_time_steps,
+                                freq='H')
+
+energysystem = solph.EnergySystem(timeindex=date_time_index)
+
+# Read data file
+filename = os.path.join(os.path.dirname(__file__), 'basic_example.csv')
+data = pd.read_csv(filename)
+
+##########################################################################
+# Create oemof object
+##########################################################################
+
+logging.info('Create oemof objects')
+# create natural gas bus
+bgas = solph.Bus(label="natural_gas")
+
+# create electricity bus
+bel = solph.Bus(label="electricity")
+
+energysystem.add(bgas, bel)
+
+# create excess component for the electricity bus to allow overproduction
+energysystem.add(solph.Sink(label='excess_bel', inputs={bel: solph.Flow()}))
+
+# create source object representing the natural gas commodity (annual limit)
+energysystem.add(solph.Source(label='rgas', outputs={bgas: solph.Flow(
+    nominal_value=29825293, summed_max=1)}))
+
+# create fixed source object representing wind power plants
+energysystem.add(solph.Source(label='wind', outputs={bel: solph.Flow(
+    actual_value=data['wind'], nominal_value=1000000, fixed=True)}))
+
+# create fixed source object representing pv power plants
+energysystem.add(solph.Source(label='pv', outputs={bel: solph.Flow(
+    actual_value=data['pv'], nominal_value=582000, fixed=True)}))
+
+# create simple sink object representing the electrical demand
+energysystem.add(solph.Sink(label='demand', inputs={bel: solph.Flow(
+    actual_value=data['demand_el'], fixed=True, nominal_value=1)}))
+
+# create simple transformer object representing a gas power plant
+energysystem.add(solph.Transformer(
+    label="pp_gas",
+    inputs={bgas: solph.Flow()},
+    outputs={bel: solph.Flow(nominal_value=10e10, variable_costs=50)},
+    conversion_factors={bel: 0.58}))
+
+# create storage object representing a battery
+storage = solph.components.GenericStorage(
+    nominal_capacity=10077997,
+    label='storage',
+    inputs={bel: solph.Flow(variable_costs=10e10)},
```

Comment:
In my example I finally changed these costs to 0.0001
Would recommend this here, too :smile: